src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
```

Comment:
I opted for a more verbose table name so it's clear we are storing just the public key (e.g. instead of `member_user_key` or something), but I am open to suggestions.